### PR TITLE
name page titles with specific name first - broader name last

### PR DESCRIPTION
**Pull request by romMidnight**
Had to change too many files, I'm not familiar with vue but probably better solution would be naming all titles through one function

https://github.com/hapijs/hapi.dev/issues/372#issuecomment-666215612

### CODE REVIEW

File: pages/plugins.vue [hunk 1]
```diff
@@ -17,7 +17,7 @@ export default {
   },
   head() {
     return {
-      title: "hapi.dev - Plugins",
+      title: "Plugins - hapi.dev",
       head() {
         return {
           title: "hapi.js - " + this.$route.params.family,
```
Comment (Nargonath) [inline pages/plugins.vue:23]: I wonder if we should not change it here too. By the way the structure is pretty odd:
```js
head() {
  return {
    head() {
       return { 
         ...
       }
    }
}
```

@jarrodyellets Do you know if that's intentional or an error slipped through?

Comment (jarrodyellets) [reply pages/plugins.vue:23]: That looks like an error to me

Comment (Nargonath) [inline pages/plugins.vue:23]: @romMidnight would you mind deleting the unneeded nesting here but keeping the `title: "Plugins - hapi.dev",` and meta attribute, please?

Comment (romMidnight) [reply pages/plugins.vue]: just did it in https://github.com/romMidnight/hapi.dev/commit/1b9fe666a48053c1e4bdf61f4431b5f8c16bf82f

**Pull request merged**